helix-term/src/commands.rs

```diff
@@ -296,6 +296,8 @@ impl MappableCommand {
         delete_selection, "Delete selection",
         delete_selection_noyank, "Delete selection without yanking",
         change_selection, "Change selection",
+        change_selection_with_indent, "Change selection and place the cursor at the appropriated indent",
```

Comment:
I am not sure what would be a good name but J think this isn't that clear. Yes this indents but the lrinarmy difference is that a newline is inserted (if lines are aelwcrez)

Reply:
I guess it is sort of  ``change_selection_at_line_start`` to follow the pattern of insert command

Reply:
Maybe `change_selection_and_auto_indent` would be clearer? Otherwise 
`change_selection_and_open_above_if_lines_selected` or
`change_selection_and_add_new_line_if_lines_selected` might be more explicit but also more verbose